Source/LinqToDB/Mapping/EntityMappingBuilder.cs

```diff
@@ -383,8 +324,7 @@ public EntityMappingBuilder<TEntity> HasIdentity<TProperty>(Expression<Func<TEnt
 				func,
 				false,
 				 _    => new IdentityAttribute(Configuration),
-				(_,a) => {},
-				a => a.Configuration);
+				(_,a) => {});
```

Comment:
I believe you can do this now (maybe not?):
```suggestion
				(_,_) => {});
```